### ISSUE TITLE
Drag and drop image from the right column

### ISSUE THREAD

**Issue by tonyo96** [fonol/anki-search-inside-add-card#27]
One of the main reasons I use this add-on is to reuse previous images. If I could simply drag and drop and image from the right to a field on the left, that would be great. 

Side notes:
* Currently when you right click on the image and select copy it doesn't copies.
* If you highlight the image then copy works as it should.
  * It copies the URL of the image file and doesn't reimports a new file into media folder which is great

**Comment by fonol** [comment]
Thanks for the suggestion, I will look into it. Actually the context menu you get when right clicking is independent of the add-on and currently not used by it in any way. So I expect the stuff to not work. But I have thought about utilizing it for some time.

**Comment by fonol** [comment]
I just checked and actually you can drag and drop images?

**Comment by tonyo96** [comment]
Drag and drop works:
* from browser window to left column
* from right column to browser window
But it does not work between left and right column.
I'm running:
* Anki: Version 2.1.11 (3cf770c7)
* the latest version of the addon
* windows 10
* and I tried disabling all the other addons but did not help
I also tried it on 2 different PCs

fonol <notifications@github.com> ezt írta (időpont: 2019. jún. 5., Sze,
16:33):
